tests/nimble_rpble_gnrc_ext/Makefile

```diff
@@ -0,0 +1,28 @@
+BOARD ?= nrf52dk
+include ../Makefile.tests_common
+
+# include shell support
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+
+# Include GNRC and RPL
+USEMODULE += gnrc_netif_single
```

Comment:
```suggestion
# Enable single interface optimization.
# Remove this if more than one interface is present
USEMODULE += gnrc_netif_single
# Include GNRC and RPL
```
Just so something is not confused (like me) on whether this is needed for nimble_rple

Reply:
sure